lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -34,26 +35,42 @@
 import org.eclipse.lemminx.dom.DOMNode;
 import org.eclipse.lemminx.services.extensions.IPositionRequest;
 import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4j.MarkupKind;
 
 public class MavenPluginUtils {
 	
-	static final String LINE_BREAK = "\n\n";
-
 	private MavenPluginUtils() {
 		// Utility class, not meant to be instantiated
 	}
 
-	public static MarkupContent getMarkupDescription(Parameter parameter) {
+	public static MarkupContent getMarkupDescription(Parameter parameter, boolean supportsMarkdown) {
+		UnaryOperator<String> toBold = supportsMarkdown ? MarkdownUtils::toBold : UnaryOperator.identity();
+		String lineBreak = supportsMarkdown ? MarkdownUtils.LINE_BREAK : "\n";
```

Comment:
Might be a bit excessive, but this could be changed to something like `String lineBreak = MarkdownUtils.getLineBreak(supportsMarkdown)` to avoid duplication of `String lineBreak = supportsMarkdown ? MarkdownUtils.LINE_BREAK : "\n";`

Reply:
Yeah, don't think its too excessive, I'll change to that.